backend/flyway/sql/V1.0.0__API.sql

```diff
@@ -0,0 +1,1013 @@
+CREATE TABLE IF NOT EXISTS "USER"
+(
+    USER_ID      uuid         not null
+        constraint "USER_PK"
+            primary key,
+    FIRST_NAME   varchar(200),
+    LAST_NAME    varchar(200) NOT NULL,
+    EMAIL        varchar(200) not null,
+    COMPANY varchar(15)  not null,
+    HIRE_DATE    date         not null,
+    SALARY       varchar
+);
+insert into "USER" (user_id, first_name, last_name, email, COMPANY, hire_date, salary)
```

Comment:
Suggestion.. add `ON CONFLICT (PK Field Name) DO NOTHING` to prevent issues when running twice the migration

Reply:
thanks @paulushcgcj , I will add that, idempotency rocks ! :)

Reply:
@paulushcgcj looks like it is handled by flyway out of the box, since all the inserts are in single transactions , so no need to worry about it :) 
https://flywaydb.org/documentation/concepts/migrations.html#transactions